<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-07T04:32:37Z" name="Default suite" finished-at="2023-02-07T04:32:38Z" duration-ms="1457">
    <groups>
    </groups>
    <test started-at="2023-02-07T04:32:37Z" name="Default test" finished-at="2023-02-07T04:32:38Z" duration-ms="1457">
      <class name="training.FirstSeleniumJavaTest">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:training.FirstSeleniumJavaTest@53045c6c]" started-at="2023-02-07T12:32:37Z" name="beforeTest" finished-at="2023-02-07T12:32:38Z" duration-ms="1328" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="TestChromeDriver()[pri:0, instance:training.FirstSeleniumJavaTest@53045c6c]" started-at="2023-02-07T12:32:38Z" name="TestChromeDriver" finished-at="2023-02-07T12:32:38Z" duration-ms="20" status="FAIL">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)
	at training.FirstSeleniumJavaTest.TestChromeDriver(FirstSeleniumJavaTest.java:38)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestChromeDriver -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:training.FirstSeleniumJavaTest@53045c6c]" started-at="2023-02-07T12:32:38Z" name="afterTest" finished-at="2023-02-07T12:32:38Z" duration-ms="93" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- training.FirstSeleniumJavaTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
